libaftermath-core/src/defs/aftermath/types/openmp/on_disk.py

```diff
@@ -250,6 +250,42 @@
 
 ################################################################################
 
+am_dsk_openmp_mutex_released = EventFrame(
+    name = "am_dsk_openmp_mutex_released",
+    entity = "on-disk OpenMP mutex released event",
+    comment = "Event signalizing release of a mutex",
+    fields = FieldList([
+        Field(
+            name = "timestamp",
+            field_type = aftermath.types.builtin.uint64_t,
+            comment = "Time of the event"),
+        Field(
+            name = "wait_id",
+            field_type = aftermath.types.builtin.uint64_t,
+            comment = "ID of the object that is bing awaited"),
```

Comment:
This type is fixed one commit further. Please fix in this original commit.

Reply:
Resolved after commits got squashed.